README.md

```diff
@@ -262,30 +262,127 @@ Parser consists of the following phases:
 ## Used technologies
 
 ### Extensions / Plugins
+main plugins:
+
+* React - https://reactjs.org/
+* Redux - https://redux.js.org/
+* Webext - https://github.com/tshaddix/webext-redux
+* styled-components - https://styled-components.com/
+* vis-network - https://visjs.github.io/vis-network/docs/network/
+* vis-timeline - https://visjs.github.io/vis-timeline/docs/timeline/
+* unique-selector - https://github.com/ericclemmons/unique-selector
+* renderjson - https://github.com/caldwell/renderjson
+* react-fontawesome - https://github.com/FortAwesome/react-fontawesome
+* Jest - https://jestjs.io/docs/en/cli.html
+* Enzyme - https://enzymejs.github.io/enzyme/
+* semver - https://github.com/npm/node-semver#readme
+* Babel - https://babeljs.io/
+* Eslint - https://eslint.org/
+* Webpack - https://webpack.js.org/
 
 ### CI
+We use CI to provide a highest quality of our code.
+
+CI checks:
+* Tests passing
+* App build without fails
+* Coverage level
+
+
+## Testing
+Currently we test our app only through the unit tests. To testing we use two main technologies: Jest + Enzyme.
```

Comment:
We believe that unit tests remain the best documentation. All React components, processing logic (helpers) and actions (such as a button click) are validated with unit tests. We use `Jest` and `Enzyme` frameworks to validate both components (React) with combination with mocked storage (Redux). 

All JS files (components & helpers) have their own tests that are placed next to the tested sources. We follow the convention:
`*.mock.js` - it is configuration containing mocks for our tests
`*.spec.jsx` - it contains unit tests

Additionally, we placed tests coverage verification in our CI. We use the `jest-coverage` tool for that. We decided to keep the coverage level at truly high levels (80 - 100%). It should enable future refactoring and code changes.

When tests are executed, then we generate the report (`test-report.xml`) file in the `build/test` folder. Moreover, there is the `coverage` directory that contains the `intex.html` file with unit tests coverage report.